src/config.js

```diff
@@ -20,5 +20,13 @@ module.exports = {
   ADDRESS_API_URL_BASE_STANDARD: `https://ws.postcoder.com/pcw/${
     process.env.ADDRESS_API_KEY || "PCW45-12345-12345-1234X"
   }/address`,
-  ADDRESS_API_URL_QUERY_STANDARD: "format=json&lines=3"
+  ADDRESS_API_URL_QUERY_STANDARD: "format=json&lines=3",
+  RNG_API_URL:
+    process.env.RNG_API_URL || "https://fsa-reference-numbers.epimorphics.net",
```

Comment:
Should this be the new FSA hosted RNG?

Reply:
You are right, I will change it. (I was using the old one until the FSA hosted service was unavailable)